enhancements/sig-policy/89-operator-policy-kind/README.md

```diff
@@ -0,0 +1,310 @@
+    - subscriptions
+    - clusterserviceversions
+    - installplans
+    - customresourcedefinitions
+    - apiservicedefinitions
+```
+
+The optional `spec.operatorGroup` section defines an
+[OperatorGroup](https://github.com/operator-framework/api/blob/master/crds/operators.coreos.com_operatorgroups.yaml).
+If not specified, and if the operator supports this InstallMode, then an
+[AllNamespaces](https://olm.operatorframework.io/docs/advanced-tasks/operator-scoping-with-operatorgroups/#targetnamespaces-and-their-relationship-to-installmodes)
+OperatorGroup will be created.
+
+The `spec.subscription` section includes fields from the spec of an [OLM
+Subscription](https://github.com/operator-framework/api/blob/master/crds/operators.coreos.com_subscriptions.yaml),
+excluding `config` and `installPlanApproval`. The `endingCSV` field allows configuration of which
+InstallPlans will be approved by the controller when the policy is set to `enforce`.
+
+The `spec.requireLatestVersion` field configures whether the "latest" version of the operator
+defined in the catalog on this cluster "should" be installed. Note that if the "latest" version is
+greater than the allowed `endingCSV`, then that version will not be automatically approved even if
+the `remediationAction` is set to enforce: instead, the policy will be NonCompliant.
+
+The `pruneOnDeletion` field allows configuration of what will be removed when the policy is deleted.
+A "recommended" setting for most workloads would be `[subscriptions, clusterserviceversions]`,
+leaving behind CRDs since they can often require extra attention to properly remove.
+
+### OperatorPolicy Status Syntax
+
+```yaml
+status:
+  compliant: Compliant
+  conditions:
+    - lastTransitionTime: '2023-02-23T19:58:52Z'
+      message: 'operator is installed as specified'
+      reason: OperatorInstalled
+      status: 'True'
+      type: Compliant
+  relatedObjects:
+    - compliant: Compliant
+      object:
+        apiVersion: operators.coreos.com/v1alpha1
+        kind: Subscription
+        metadata:
+          name: my-operator
+          namespace: own-namespace
+      reason: Resource created
+```
+
+As required for all policies, there is a `status.compliant` field for the overall status. There is
+also a `status.relatedObjects` field with the same format as the one in ConfigurationPolicy, for the
+OLM objects related to this operator installation.
+
+New (compared to other policies) here is a `status.conditions` field compatible with `oc wait
+--for=...`. At the very least, this will include a condition of type "Compliant", but additional
+fields for internal state may be exposed, for example when waiting for a ClusterServiceVersion after
+approving an InstallPlan.
+
+### User Stories
+
+#### Story 1
+
+As a policy user, I want to install an operator of a specific version, from a specific source, on my
+managed clusters. I should be able to view the health of these installations through policies.
```

Comment:
In my opinion of what a user story should be, it doesn't make sense to pull the health reporting out into a separate story. I do agree it could help feature development, but I didn't want to artificially split it up here.